src/pages/docs/tutorials/tutorial-hello-enact/basics/index.md

```diff
@@ -56,13 +56,13 @@ The first step is to `import` our dependencies for this component. The [`import`
 ```js
```

Comment:
The line 54, ES6 link broken

Reply:
It seems that the link ES6's address is changed recently.
Fixed.